src/dotnet/CommandResolution/ProjectToolsCommandResolverPolicy.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.CommandResolution
+{
+    public class ProjectToolsCommandResolverPolicy : ICommandResolverPolicy
+    {
+        public CompositeCommandResolver CreateCommandResolver()
+        {
+            var defaultCommandResolverPolicy = new DefaultCommandResolverPolicy();
+            var compositeCommandResolver = defaultCommandResolverPolicy.CreateCommandResolver();
+            var packagedCommandSpecFactory = new PackagedCommandSpecFactory();
+
+            compositeCommandResolver.AddCommandResolver(new ProjectToolsCommandResolver(packagedCommandSpecFactory));
```

Comment:
I don't understand why this isn't added to the `DefaultCommandResolverPolicy()`.  Why do we need a whole new ProjectToolsCommandResolverPolicy?